doc/nrf/app_dev/device_guides/working_with_nrf/nrf54h/ug_nrf54h20_gs.rst

```diff
@@ -317,11 +336,17 @@ It uses the ``nrf54h20dk/nrf54h20/cpuapp`` board target.
 To build and program the sample to the nRF54H20 DK, complete the following steps:
 
 1. Connect the nRF54H20 DK to your computer using the **DEBUGGER** port on the DK.
-#. Open nRF Connect for Desktop, navigate to the Toolchain Manager, select the v2.7.99-cs2 toolchain, and click the :guilabel:`Open terminal` button.
+#. Open nRF Connect for Desktop, navigate to the Toolchain Manager, select the v2.8.0 toolchain, and click the :guilabel:`Open terminal` button.
 #. In the terminal window, navigate to the :file:`zephyr/samples/sysbuild/hello_world` folder containing the sample.
-#. Build the sample for application and radio cores by running the following command::
+#. Build the sample for application and radio cores by running the following commands:
+
+   * For Engineering C (DKs from v0.8.0 to 0.8.2)::
```

Comment:
Wrong revisions, it should be
`For Engineering C (DKs 0.8.3 and later revisions)`